src/native-stack/utils/ScreenInfoContext.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { NativeStackNavigationOptions } from '../types';
+
+type ScreenInfoType = {
+  options: NativeStackNavigationOptions;
+};
+
+const ScreenInfoContext = React.createContext<ScreenInfoType>({
+  options: {},
+});
+
```

Comment:
Nitpick: If we're gonna go with context let's work on naming. I don't have any suggestion for now, unfortunately, but will work on it 🤞🏻

Reply:
Update & remainder for me: We talked and I've suggested to pass this information alongside `enabled`, `isNativeStack` as a prop instead of using context.

Reply:
Done! https://github.com/software-mansion/react-native-screens/pull/1915/commits/b2d876ac20a3972688451fdf3f9b62c12086f6c2